config/webpack.js

```diff
@@ -492,10 +492,17 @@ module.exports = (env) => {
         fail_build: true,
       };
 
-      if (process.env.TRAVIS === 'true') {
+      if (process.env.TRAVIS === 'true' || process.env.CI === 'true') {
         Object.assign(packOptions, {
-          branch: process.env.TRAVIS_PULL_REQUEST_BRANCH || process.env.TRAVIS_BRANCH,
-          commit: process.env.TRAVIS_PULL_REQUEST_SHA || process.env.TRAVIS_COMMIT,
+          branch:
+            process.env.TRAVIS_PULL_REQUEST_BRANCH ||
+            process.env.TRAVIS_BRANCH ||
+            process.env.PR_BRANCH ||
+            process.env.PUSH_BRANCH,
```

Comment:
You're setting PR_BRANCH but not PUSH_BRANCH?

Reply:
No so github makes it hard to get branch names. 

In the event of a push to master, we can get the branch name via `echo "PUSH_BRANCH=${GITHUB_REF##*/}" >> $GITHUB_ENV`. The env var GITHUB_REF has the branch name in it but has the form refs/head/branch-name. This extracts it out.

In the event of a pull request we can get it directly via the github context item, `${{ github.event.pull_request.head.ref }}`.

Reply:
If we move to seperate actions for pr's and pushes to master we wont need to do this so I might do that. It's what deliphi had set up initially so I will use some of the work he had done but split it up into a few PR's starting with pull requests.